projects/collection-editor-library/src/lib/components/editor/editor.component.ts

```diff
@@ -829,15 +830,15 @@ export class EditorComponent implements OnInit, OnDestroy, AfterViewInit {
         isReadOnlyMode: _.get(this.editorConfig, 'config.isReadOnlyMode'),
         unitIdentifier: _.get(this.editorConfig, 'context.unitIdentifier'),
         correctionComments: _.get(this.editorConfig, 'context.correctionComments'),
-        mode: mode,
+        mode,
         editableFields: _.get(this.editorConfig, 'config.editableFields')
       };
     }
 
     this.questionComponentInput = {
       ...this.questionComponentInput,
       questionSetId: this.collectionId,
-      questionId: questionId,
+      questionId,
```

Comment:
@vinukumar-vs 
I did not remove this code. I have fixed tslint syntax error : `Expected property shorthand in object literal ('{questionId}').tslint(object-literal-shorthand)`

In ES6, We can use {questionId} instead of {questionId: questionId}.
Ref. link : [https://eslint.org/docs/rules/object-shorthand#rule-details](https://eslint.org/docs/rules/object-shorthand#rule-details)